src/ui/actions/PlaylistsActions.ts

```diff
@@ -123,6 +123,7 @@ export const addTracks = async (_id: string, tracksIds: string[], isShown?: bool
     const playlist = await app.models.Playlist.findOneAsync({ _id });
     const playlistTracks = playlist.tracks.concat(tracksIds);
     await app.models.Playlist.updateAsync({ _id }, { $set: { tracks: playlistTracks } });
+    await refresh();
```

Comment:
I noticed this bug as well but I am not sure this is the "right" way to fix it, I'll need to investigate a tiny bit before accepting this.